src/modules/generateRandomNumber.js

```diff
@@ -8,6 +8,19 @@
  */
 function generateRandomNumber() {
   /* Write your code here */
+  const availableNums = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const result = [];
+
+  const getRandomN = () => Math.floor(Math.random() * availableNums.length);
+
+  result.push(availableNums.splice(getRandomN(), 1)[0]);
+  availableNums.unshift(0);
```

Comment:
The use of `unshift` here is not clear. If you need to add `0` to the list of available numbers, why not include it in the original array? Also, it's not clear why `0` is added after the first number is selected. This could lead to confusion for other developers reading your code.